webpack/assets/javascripts/foreman_users.test.js

```diff
@@ -70,3 +70,31 @@ describe('taxonomyAdded', () => {
     expect($('#user_default_organization_id option').length).toEqual(1);
   });
 });
+
+describe('users Password', () => {
+  xit('should toggle password field', () => {
```

Comment:
can you just add a comment here why we're skipping it? so we know to enable it once jest fixes the issues it has?